validation-tool-backend/src/review/review.interface.ts

```diff
@@ -44,3 +44,20 @@ export interface ReviewReportItem {
   assigned_items: number;
   not_answered: number;
 }
+
+export interface ReviewProgressDto {
+  run_id: number;
+  n_revd: number;
+  n_tot: number;
+  pct_complete: number;
+}
+
+export interface ReviewStatistics {
```

Comment:
doesn't look used

Reply:
Since i fixed the last item, now it's going to be used.